app/question_list_helper.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="description" content="SUMO real time support app for FirefoxOS" />
+    <meta name="viewport" content="width=device-width" />
+
+    <title>My Questions - BuddyUP</title>
+
+    <link rel="stylesheet" type="text/css" href="css/building_blocks/layout.css" media="screen" />
+    <link rel="stylesheet" type="text/css" href="css/building_blocks/headers.css" media="screen" />
+    <link rel="stylesheet" type="text/css" href="css/building_blocks/lists.css" media="screen" />
+    <link rel="stylesheet" type="text/css" href="css/building_blocks/tabs.css" media="screen" />
+
+    <link rel="stylesheet" type="text/css" href="css/fira.css" media="screen" />
+    <link rel="stylesheet" type="text/css" href="css/gaia-icons.css" media="screen" />
+    <link rel="stylesheet" type="text/css" href="css/buddyup.css" media="screen" />
+
+    <!-- localization -->
+    <link rel="prefetch" type="application/l10n" href="data/locales.ini" />
+    <link rel="prefetch" type="application/l10n" href="data/date/date.ini" />
+    <script src="js/libs/l10n.js" defer></script>
+    <script src="js/libs/l10n_date.js" defer></script>
+  </head>
+  <body class="skin-organic">
+    <section role="region" class="vbox fit">
+      <header>
+        <h1 id="main_heading">My Questions</h1>
+        <a href="index.html" id="back" class="back-link" data-icon="back-light">
+          <span class="visuallyhidden">Back</span>
+        </a>
+        <a href="profile.html" id="settings" class="settings-icon" data-icon="settings">
+          <span class="visuallyhidden">Settings</span>
+        </a>
+      </header>
+      <main class="fit scroll">
+        <section class="tabbed-section" role="region">
+          <ul role="tablist" class="bb-tablist" data-type="filter">
+            <li id="new" role="presentation">
+              <a id="tab1" href="#panel_new" role="tab" aria-controls="panel_new" aria-selected="true">New</a>
+            </li>
+            <li id="active" role="presentation">
+              <a id="tab2" href="#panel_active" role="tab" aria-controls="panel_active" aria-selected="false">Active</a>
+            </li>
+          </ul>
+
+          <div id="panel_new" class="bb-tabpanel" role="tabpanel" aria-labelledby="tab1" aria-hidden="false">
+            <section id="myquestions" class="questions all" data-all="true"></section>
+            <button id="load-more" class="button-gray hide">Load more…</button>
+          </div>
+
+          <div id="panel_active" class="bb-tabpanel" role="tabpanel" aria-labelledby="tab2" aria-hidden="true">
+            <section data-type="budyup-list">
```

Comment:
It should be `buddyup-list` but I didn't want to add noise to this patch by fixing it.